### ISSUE TITLE
Expose SuggestedLocation to library consumers

### ISSUE THREAD

**Issue by grote** [schildbach/public-transport-enabler#138]
Would you accept a PR that makes the `SuggestedLocation`s in `SuggestedLocationResult` public?

The use case is advanced sorting of auto-completion results that should take the priority value returned by the APIs into account.

**Comment by schildbach** [comment]
I don't understand, the SuggestLocationsResult constructor already sorts the list of suggestions, and the natural ordering is by priority.

**Comment by grote** [comment]
Yes, the returned list of locations is sorted by priority. That is nice and what I have been using so far. However, if you want to do custom sorting based on other criteria such as distance to current location, having the actual priority would be nice so it can be taken into account.

grote/Transportr#257 is an example for where this can be useful.

**Comment by schildbach** [comment]
Ok you convinced me. I'm fine with making that field public.
